chapter10/axp/ex02

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env perl
+
+use 5.014;
+use warnings;
+use Benchmark qw(timethese);
+
+chdir( $ARGV[0] // "~" );
+
+timethese( 5, { naive => \&naive, schwartzian => \&schwartzian } );
```

Comment:
Strange. Either we're working from slightly different library versions, or I ran it against a larger directory than you, and it took that into account. I get the same message if I reduce it much further, though.